server/src/main/java/org/apache/iotdb/db/engine/StorageEngine.java

```diff
@@ -303,9 +303,7 @@ public void insert(InsertPlan insertPlan)
   public void syncCloseAllProcessor() {
     logger.info("Start closing all storage group processor");
     for (StorageGroupProcessor processor : processorMap.values()) {
-      processor.waitForAllCurrentTsFileProcessorsClosed();
-      //TODO do we need to wait for all merging tasks to be finished here?
-      processor.closeAllResources();
```

Comment:
Don't we have to close the file resources? Should we move it to syncCloseAllTsFileProcessors() instead of directly deleting it?

Reply:
It's already be closed in closeUnsealedTsFileProcessor() when the close call back